data/json/npcs/talk_tags.json

```diff
@@ -751,7 +759,8 @@
         "Quiet down over there!",
         "Did you hear someone talking?",
         "Who said that?",
-        "Who's there?"
+        "Who's there?",
+		"Shhh.. I've heard something"
```

Comment:
Should read: "Shhh... I heard something." - three dots in an ellipsis, no contraction, and full stop at end.